gradle/test.gradle

```diff
@@ -3,5 +3,12 @@ tasks.withType(Test) {
         events 'FAILED'
         exceptionFormat 'full'
     }
+    doFirst {
+      // http://openjdk.java.net/jeps/223
+      if (System.getProperty("java.specification.version") == "9") {
+        jvmArgs += [
+          '--add-opens', 'java.base/java.lang=ALL-UNNAMED'
```

Comment:
I believe we need another option too `--add-modules=ALL-SYSTEM` ???, see 
```
:eclipsePlugin-junit:testPicked up _JAVA_OPTIONS: -Xmx2048m -Xms512m
java.lang.NoClassDefFoundError: jdk/internal/reflect/GeneratedSerializationConstructorAccessor1
	at jdk.internal.reflect.GeneratedSerializationConstructorAccessor1.newInstance(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:488)
	at java.base/java.io.ObjectStreamClass.newInstance(ObjectStreamClass.java:1009)
	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2035)
	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1559)
	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:423)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:105)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:62)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:62)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:67)
Caused by: java.lang.ClassNotFoundException: jdk.internal.reflect.GeneratedSerializationConstructorAccessor1
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:466)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:563)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:496)
	... 10 more
 FAILED
```

Reply:
JVM argument is enough. I found that `+=` in `gradle/test.gradle` introduced this problem, I will use `=` instead.